articles/openshift/oshift-how-expand-storage.md

```diff
@@ -0,0 +1,76 @@
+---
+title: How to expand persistent volumes | UKCloud Ltd
+description: Shows how to expand persistent volumes using the OpenShift CLI
+services: openshift
+author: Kieran O'Neill
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Expand persistent volumes
+toc_fullpath: How To/oshift-how-expand-storage.md
+toc_mdlink: oshift-how-expand-storage.md
+---
+
+# How to expand persistent volumes
+
+## Overview
+
+This is a step-by-step guide on expanding persistent storage volumes from the command line. In order to follow this guide your OpenShift cluster needs to be v3.11 or later, you need access to the OpenShift command line tools and you need sufficient permissions to edit the PersistentVolumeClaim that is bound to the target storage.
+
+## Steps
```

Comment:
Probably need to add a section here about verifying its possible to expand.

```suggestion
## Pre-checks
Before you can carry out this task you need to confirm the storage class can be expanded. We currently support this on OpenStack Cinder backed persistent volumes only. To verify if your storage class is based on this carry out the following checks.

In this example we're verifying that the tier2 storage class is able to be expanded.

```oc describe sc tier2
Name:                  tier2
IsDefaultClass:        Yes
Annotations:           storageclass.beta.kubernetes.io/is-default-class=true
Provisioner:           kubernetes.io/cinder
Parameters:            availability=nova,type=TIER2
AllowVolumeExpansion:  True
MountOptions:          <none>
ReclaimPolicy:         Delete
VolumeBindingMode:     Immediate
Events:                <none>```

You can see from the Provisioner that this is OpenStack Cinder based storage, and from the AllowVolumeExpansion: True parameter that this storage class has expansion enabled.

With this example we are able to proceed.

## Steps
```